### DISCUSSION TITLE
Modify album and tracks simultaneously

### DISCUSSION THREAD

**Discussion by dschrempf** [beetbox/beets#4172] [Q&A] [answered]
Hi!

At the moment, if I want to change the compilation flag for an album, I have to to

```
beet modify album comp=1
beet modify -a new-path-to-album comp=1
```
where `new-path-to-album` is usually `Compilations/album`.

Is it possible to set flags for an album and the tracks simultaneously?

**Comment by sampsyo** [comment, marked as answer]
Hmm, that's odd—I thought `beet modify -a` all by itself would do just fine. Modifying an album-level field should propagate changes to the items in the album. This happens right about here:
https://github.com/beetbox/beets/blob/3c7853712f83aafb3a9aa7b1c2943b45858e660f/beets/library.py#L1337-L1339

Any chance that does happen to work all by itself?

**Reply by dschrempf** [reply]
Oh, I didn't know that. It is not clear from the documentation. Maybe I can add a sentence about that?

**Reply by sampsyo** [reply]
Sounds good!
